### ISSUE TITLE
46.beta - Taskbar focus indicator rectangle overlaps rounded running application highlights

### ISSUE THREAD

**Issue by tekstryder** [home-sweet-gnome/dash-to-panel#2044]
Arch Linux | Kernel 6.7.6
Gnome-shell | Mutter 46.beta
Wayland (gnome-shell --no-x11)
Dash-to-Panel Master+https://github.com/home-sweet-gnome/dash-to-panel/pull/2028

![dash-to-panel-focus-rectangular-vs-round](https://github.com/home-sweet-gnome/dash-to-panel/assets/11696773/b97e017b-5811-4585-8c61-352e607ed4b6)

So, either the focus indicator highlight should be rounded to match (preferable), or the running app indicators should be rectangular to match.

**Comment by darkblaze69** [comment]
Bubbles started started since [style: Refactor the button and entry drawing css](https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/a917644fad589f28bf66fcc4cd8b34bd5772cdfc)

**Comment by City-busz** [comment]
https://github.com/home-sweet-gnome/dash-to-panel/pull/2058 fixes the problem.

**Comment by tekstryder** [comment]
> #2058 fixes the problem.

Verified fixed applied atop Master+https://github.com/home-sweet-gnome/dash-to-panel/pull/2028 branch

I'll close this issue when this is merged and the next release is tagged.

Thanks!

p.s. I was kinda starting to grow fond of the bubbles, and had simply disabled the highlight as a workaround.
